File: divisible sum pairs/Divisible Sum Pairs.php
<?php

function divisibleSumPairs($n, $k, $ar)
{
    $count = 0;
    for ($i = 0; $i < $n; $i++) {
        for ($j = $i + 1; $j < $n; $j++) {
            if (($ar[$i] + $ar[$j]) % $k == 0) {
                $count++;
            }
        }
    }
    return $count;
}

$input = explode(" ", readline());
$n = intval($input[0]);
$k = intval($input[1]);
$ar = array_map('intval', explode(" ", readline()));
$result = divisibleSumPairs($n, $k, $ar);
echo $result;

?>

File: divisible sum pairs/Divisible Sum Pairs.py
def divisible_sum_pairs(n, k, ar):
    count = 0

    for i in range(n):
        for j in range(i + 1, n):
            if (ar[i] + ar[j]) % k == 0:
                count += 1

    return count


if __name__ == "__main__":
    n, k = map(int, input().split())
    ar = list(map(int, input().split()))

    result = divisible_sum_pairs(n, k, ar)
    print(result)
